components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/EventManagementUtil.java

```diff
@@ -37,10 +37,14 @@ public static StreamDefinition constructStreamDefinition(String syncId, org.wso2
 
         List<Attribute> attributes = new ArrayList<Attribute>();
         if (inStreamDefinition.getMetaData() != null) {
-            attributes.addAll(inStreamDefinition.getMetaData());
+            for (Attribute attr : inStreamDefinition.getMetaData()) {
+                attributes.add(new Attribute("meta_" + attr.getName(), attr.getType()));
```

Comment:
@grainier 
Can you change these to string constants and refer to the constant from here? There already should exist some of these constants in these components.